package.json

```diff
@@ -60,13 +60,13 @@
     "pug": "^3.0.2",
     "request": "^2.88.2",
     "response-time": "^2.3.2",
-    "selectize": "0.12.4",
```

Comment:
I'm so glad to see the back of this, to be honest :) It's had a very long standing dependency on something with a security hole... something I think Tom Select's fork has fixed, yay!